packages/experimental/src/components/AboutModal/_about-modal.scss

```diff
@@ -13,131 +13,111 @@
-    margin-right: $spacing-04;
-    object-fit: contain;
-  }
-  .#{$pkg-prefix}-about-modal-tab-content-flex {
-    display: flex;
-    align-items: center;
-    height: inherit;
-    &.#{$pkg-prefix}-about-modal-tab-content-version-flex {
-      flex-direction: column;
-      align-items: flex-start;
-      justify-content: center;
-    }
-  }
-  .#{$pkg-prefix}-about-modal-tab-container {
-    position: relative;
-    /* stylelint-disable-next-line carbon/layout-token-use */
-    bottom: carbon--rem(20px);
-  }
+.#{$pkg-prefix}-about-modal .bx--modal-header {
+  grid-row: auto;
+  margin-bottom: 0;
+  padding: 0 20% $spacing-03 $spacing-05;
 }
 
-.#{$pkg-prefix}-about-modal.#{$pkg-prefix}-about-modal-with-tabs
-  .bx--modal-content.exp-about-modal-content {
-  margin-bottom: $layout-06;
+.#{$pkg-prefix}-about-modal-title {
+  @include carbon--type-style('productive-heading-04');
 }
 
-.#{$pkg-prefix}-about-modal.#{$pkg-prefix}-about-modal-with-tabs
-  .bx--tab-content {
-  position: relative;
-  /* stylelint-disable-next-line carbon/layout-token-use */
-  bottom: carbon--rem(20px);
-  height: $layout-05;
+.#{$pkg-prefix}-about-modal-content {
+  @include carbon--type-style('body-short-02');
+
+  grid-row: auto;
   min-height: $layout-05;
-  padding: 0;
+  margin-bottom: 0;
+  padding: 0 20% 0 $spacing-05;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+.#{$pkg-prefix}-about-modal-with-tabs .#{$pkg-prefix}-about-modal-content {
+  min-height: calc(#{$layout-05} + #{$spacing-08});
 }
 
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-content,
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-legal-text,
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-copyright-text {
-  color: $text-01;
+.#{$pkg-prefix}-about-modal-body-content {
+  margin-bottom: $spacing-09;
 }
 
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-legal-text,
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-copyright-text {
+.#{$pkg-prefix}-about-modal-with-tabs .#{$pkg-prefix}-about-modal-body-content {
+  // stylelint-disable-next-line carbon/layout-token-use
+  margin-bottom: calc(#{$spacing-09} + #{$spacing-08});
+}
+
+.#{$pkg-prefix}-about-modal-links-container {
+  margin-top: $spacing-05;
+}
+
+.#{$pkg-prefix}-about-modal-links-container a + a {
+  margin-left: $spacing-03;
+  padding-left: $spacing-03;
+  border-left: 1px solid $text-01;
+}
+
+.#{$pkg-prefix}-about-modal-legal-text,
+.#{$pkg-prefix}-about-modal-copyright-text {
   margin-top: $spacing-07;
   color: $text-03;
 }
 
-.#{$pkg-prefix}-about-modal .#{$pkg-prefix}-about-modal-copyright-text {
+.#{$pkg-prefix}-about-modal-copyright-text {
   margin-top: $spacing-05;
 }
 
-.#{$pkg-prefix}-about-modal .bx--modal-header,
-.#{$pkg-prefix}-about-modal .bx--modal-content {
-  grid-row: auto;
-  padding: 0 $spacing-05;
+.#{$pkg-prefix}-about-modal .bx--modal-footer {
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  height: rem(72px);
```

Comment:
This was 64px before. Why the change to 72px? To my eye, 64px looks a closer match to the design drawing.

Reply:
The design spec spacing section calls for 2x padding of 16px plus spacing of 4px plus 2 lines of text at 18px = 72px. At the moment the top and bottom padding is too small, but the flex box centring makes this unobvious.